src/Runner.Common/Terminal.cs

```diff
@@ -168,6 +170,10 @@ public void WriteSection(string message)
                 Console.WriteLine($"# {message}");
                 Console.WriteLine();
             }
+            else
+            {
+                Trace.Info($"# {message}");
```

Comment:
So that we still print to stdout through the `ConsoleListener` even if Terminal.cs is silent